Sources/Markdown/Interpretive Nodes/Aside.swift

```diff
@@ -159,7 +159,17 @@ public struct Aside {
             self.rawValue = rawValue
         }
     }
-    
+
+    /// Determines the permissiveness of aside-tag parsing when using ``init(_:tagRequirement:)``.
+    public enum TagRequirement: Equatable {
+        /// Only allow asides with a single-word aside tag, such as `Warning:` or `Important:`
+        case requireSingleWordTag
+        /// Require a aside tag, but allow it to be multiple words, such as `See Also:`
+        case requireAnyLengthTag
+        /// Convert all block-quotes into asides, treating asides with no kind tag as ``Aside/Kind/note`` asides.
+        case tagNotRequired
```

Comment:
👍 I like having the 3rd "no requirement" be an enum case like this rather than a `nil` value because it allows us to document the case and make it more explicit at the call site.